core/src/main/resources/error/error-classes.json

```diff
@@ -55,7 +83,12 @@
     "sqlState" : "22023"
   },
   "INVALID_JSON_SCHEMA_MAPTYPE" : {
```

Comment:
While we're here, can you also update: `INVALID_JSON_SCHEMA_MAPTYPE` -> `INVALID_JSON_MAP_KEY_TYPE`? The original class name is not very descriptive.